### ISSUE TITLE
Debug of variables

### ISSUE THREAD

**Issue by giovannicandido** [lihaoyi/utest#63]
It would be nice to have a way to debug variables, `println` do not work.
In javascript land, you constant doens't know the content of variable or object until you look at. The browser ofers the console.debug that print a nice representation of any object.

I'm constant struggling to test things like that:

```
utest.AssertionError: assert(ndom.children().length == 0);
[info]                                       		ndom: org.scalajs.jquery.JQuery = [object Object]
```
Why this assertion don't work? The length of the children is a object instead of Int? What is the value?
One quick and dirty way is to log the the output of lenght and see what it has but it is not possible to do that. 

The most strange thing is that this test pass sometimes, so the problem here is that utest is unable to provide the content of the variable in the test result?

**Comment by giovannicandido** [comment]
jQuery is a good example. The static type library ofer static analysis to some of the interfaces, but not all of them. And the interfaces of traversable objects are dificult if no impossible to provide.
As a example:
```
val ndom = jQuery("#anydiv")
ndom.offset().top // compile error
// The solution is:
val position = ndom.offset().asInstanceOf[js.Dynamic]
assert(position.top == 0);
```
But wait, what is the content of top if the assert fails? [object Object] is not a good answer

**Comment by lihaoyi** [comment]
Yeah we need better debugging macros.

**Comment by lihaoyi** [comment]
Fixed in the WIP `test` branch targeting 0.4.0
